test/src/in_language_tests/test_programs/primitive_conversions_b256_inline_tests/src/main.sw

```diff
@@ -11,10 +11,10 @@ fn b256_try_from_bytes() {
         initial_bytes.push(51u8);
         i += 1;
     }
-    let res = b256::try_from(initial_bytes);
-    let expected = 0x3333333333333333333333333333333333333333333333333333333333333333;
-
-    assert(res.unwrap() == expected);
+    let res1 = b256::try_from(initial_bytes);
+    let expected1 = 0x3333333333333333333333333333333333333333333333333333333333333333;
+    assert(res1.is_some());
```

Comment:
doesnt the unwrap imply that res1 is Some?

Reply:
It is better to fail intentionally than fail on unwrapping `None`. Fail fast and fail early.